test/e2e/suites/unmanaged/unmanaged_functional_test.go

```diff
@@ -647,7 +647,7 @@ var _ = ginkgo.Context("[unmanaged] [functional]", func() {
 			Expect(shared.ReleaseAddress(e2eCtx, allocationID)).To(BeTrue())
 
 			ginkgo.By("Detaching Internet gateway")
-			Expect(shared.DetachInternetGateway(e2eCtx, igwID, vpcID)).To(BeTrue())
+			Eventually(shared.DetachInternetGateway(e2eCtx, igwID, vpcID), 60*time.Second).Should(BeTrue())
```

Comment:
Do we need 60s wait time? Does the detaching takes that long to finish? Or can this be reduces to 20-30s maybe. Wdyt?

Reply:
@Ankitasw I honestly don't know. I've only seen it fail once and I don't have the time required on that test. I can change it to 30 seconds. My understanding was that Gomega would pass if the condition was meet before the timeout.

Reply:
yes, but it wont pass if it doesn't meet till 60s, so in worst case this test case will add additional 60s delay, so I think it would be better to keep it within 30s.